### ISSUE TITLE
pastebinit fails with bad API request

### ISSUE THREAD

**Issue by ProfessorKaos64** [ValveSoftware/SteamOS#449]
Fails:
```
desktop@steamos:~/SteamOS-Tools-Packaging/sprunge$ cat ~/test.txt | pastebinit 
Bad API request, invalid api_dev_key
````

Works:
```
desktop@steamos:~/SteamOS-Tools-Packaging/sprunge$ cat ~/test.txt | pastebinit -b http://paste.debian.net
http://paste.debian.net/
```

Bug report upstream:
https://www.mail-archive.com/debian-bugs-dist@lists.debian.org/msg1303214.html

I could recompile the package you have to include a default destination link to debian's, but I would prefer Valve make the call. If you check the upstream bug, it seems that http://pastebin.com, the default paste service,became closed to anonymous pasting. It is quite easy to divert lsb_release to detect SteamOS. Since SteamOS does not have it's own destination origin, I did not make any changes myself.

The uptream patches seem to be present, but since the default PB is closed to anonymous pasting, this is an issue.

**Comment by not83** [comment]
Is pastebinit included in SteamOS by default? If not couldn't this issue be closed. I would understand if pastebinit was included by default however

**Comment by ProfessorKaos64** [comment]
@not83, yes or I would not have to bring this up and be able to supersede the Jessie package in my SteamOS-Tools package. 

repo.steampowered.com/steamos/pool/main/p/pastebinit/pastebinit_1.4-4+bsos3_all.deb

**Comment by sharkwouter** [comment]
@not83 It is not installed by default, but available in the repo.

**Comment by ProfessorKaos64** [comment]
Here is some back story if Valve chooses to fix the package: http://mewbies.com/pastebinit_how_to_pastebin_commnad_line_output_from_terminal_to_pastebin.htm
